modules/operator-upgrade.adoc

```diff
@@ -46,6 +46,55 @@ To update {productname} from one minor version to the next, for example, 3.4 ->
 
 For `z` stream upgrades, for example, 3.4.2 -> 3.4.3, updates are released in the major-minor channel that the user initially selected during install. The procedure to perform a `z` stream upgrade depends on the `approvalStrategy` as outlined above. If the approval strategy is set to `Automatic`, the Quay Operator will upgrade automatically to the newest `z` stream. This results in automatic, rolling Quay updates to newer `z` streams with little to no downtime. Otherwise, the update must be manually approved before installation can begin.
 
+[id="upgrading-postgresql-databases"]
+=== Updating {productname} from 3.8 -> 3.9
+
+When updating {productname} 3.8 -> 3.9, the Operator automatically upgrades the existing PostgreSQL databases for Clair and {productname} from version 10 to version 13. 
+
+[IMPORTANT]
+====
+* This upgrade is irreversible. It is highly recommended that you upgrade to PostgreSQL 13. PostgreSQL 10 * will have its final release in May, 2024. Afterwards, it will be unsupported. 
+* By default, {productname} is configured to remove old persistent volume claims (PVCs) from PostgreSQL 10. To disable this setting and backup old PVCs, you must set `POSTGRES_UPGRADE_RETAIN_BACKUP` to `True` in your `config.yaml` file. 
+====
+
+.Prerequisites 
+
+* You have installed {productname} 3.8 on {ocp}. 
+* 100 GB of free, additional storage.
++
+During the upgrade process, additional persistent volume claims (PVCs) are provisioned to store the migrated data. This helps prevent a destructive operation on user data. The upgrade process rolls out PVCs for 50 GB for both the {productname} database upgrade, and the Clair database upgrade. 
+
+.Procedure
+
+. Optional. Back up your old PVCs from PostgreSQL 10 by setting `POSTGRES_UPGRADE_RETAIN_BACKUP` to `True` your `config.yaml` file. For example:
++
+[source,yaml]
+----
+POSTGRES_UPGRADE_RETAIN_BACKUP: True
+----
+
+. In the {ocp} Web Console, navigate to *Operators* -> *Installed Operators*. 
```

Comment:
Do we need to document how to do it via the command line?

Reply:
Yes. I think that would be helpful. I can compile some steps that I will assume are incorrect, and request that you and @SeanZhao-redhat review them. Thanks.

Reply:
lgtm